examples/apollo_server/test/tests.ts

```diff
@@ -0,0 +1,74 @@
+import * as dns from 'node:dns';
+dns.setDefaultResultOrder('ipv4first');
+
+import { Test } from 'azle/test';
+
+export function getTests(canisterId: string): Test[] {
+    const origin = `http://${canisterId}.localhost:8000`;
+
+    return [
+        {
```

Comment:
It troubles me a little bit that there are only two tests. I understand that we don't have time to test every facet of every api, but this is two tests testing essentially the same functionality just on two different lists